docs/single-process-images.md

```diff
@@ -23,6 +27,8 @@ Currently built with the following plugins:
 - [pulp_python](https://docs.pulpproject.org/pulp_python/)
 - [pulp_rpm](https://docs.pulpproject.org/pulp_rpm/)
 
+Note that the multi-process image `pulp` can also function in the same way as this image.
```

Comment:
Maybe this should be specified in the page on the mutli-process images instead?

Reply:
yeah .. it belongs in the multiprocess image docs